modin/engines/ray/pandas_on_ray/io.py

```diff
@@ -52,6 +52,7 @@ def _read_parquet_columns(path, columns, num_splits, kwargs):  # pragma: no cove
     import pyarrow.parquet as pq
 
     df = pq.read_pandas(path, columns=columns, **kwargs).to_pandas()
+    df = df[columns]
```

Comment:
Just delete this. It will be fixed when the resolution for #636 is added.

Reply:
sounds good! This will be blocked on #638